homeassistant/components/ios/sensor.py

```diff
@@ -1,17 +1,26 @@
 """Support for Home Assistant iOS app sensors."""
+from __future__ import annotations
+
 from homeassistant.components import ios
-from homeassistant.components.sensor import SensorEntity
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
 from homeassistant.const import PERCENTAGE
 from homeassistant.core import callback
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.icon import icon_for_battery_level
 
 from .const import DOMAIN
 
-SENSOR_TYPES = {
-    "level": ["Battery Level", PERCENTAGE],
-    "state": ["Battery State", None],
-}
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key="level",
+        name="Battery Level",
+        unit_of_measurement=PERCENTAGE,
```

Comment:
Would it make sense to add `device_class=sensor.DEVICE_CLASS_BATTERY`?

Reply:
See https://github.com/home-assistant/core/pull/54317#discussion_r685559271

Reply:
That would be a functional change that is out of scope for this PR.